bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/html/HtmlUtil.scala

```diff
@@ -0,0 +1,42 @@
+package edu.gemini.itc.web.html
+
+import edu.gemini.itc.shared.SourceDefinition
+import edu.gemini.spModel.target._
+
+/**
+ * HTML Utils
+ * Move replacement code for Java HtmlPrinter here.
+ */
+object HtmlUtil {
+
+  // string representation for the spectral distribution
+  def sourceDistributionString(sd: SourceDefinition): String = sd.distribution match {
+
+    case EmissionLine(wavelength, width, flux, cont) =>
+      s"n emission line, at a wavelength of $wavelength, and with a width of $width.\n" +
+      s"It's total flux is $flux on a flat continuum of flux density $cont."
+
```

Comment:
Thanks to squants this automatically takes the units into account and produces nicely formatted output like for example: 
```
The z = 0.0 extended source is an emission line, at a wavelength of 0.76 µm, and with a width of 500.123 km/s.
It's total flux is 5.123E-17 erg/s/cm² on a flat continuum of flux density 1.0E-17 W/m²/µm.
```